react/providers/CozyTheme/index.jsx

```diff
@@ -12,6 +12,9 @@ import CozyThemeWithQuery from './CozyThemeWithQuery'
 
 export const CozyThemeContext = createContext()
 
+/**
+ * @returns {{ type: 'light'|'dark', variant: 'normal'|'inverted', isLight: boolean, isDark: boolean }} An array with a state value and a function to update it.
```

Comment:
`An array with a state value and a function to update it` copier/coller malencontreux non ?